kubernetes/client/apis/custom_objects_api.py

```diff
@@ -1657,7 +1657,7 @@ def patch_cluster_custom_object_with_http_info(self, group, version, plural, nam
 
         # HTTP header `Content-Type`
         header_params['Content-Type'] = self.api_client.\
-            select_header_content_type(['application/merge-patch+json', 'application/json-patch+json'])
+            select_header_content_type(['application/json-patch+json', 'application/merge-patch+json'])
```

Comment:
This is a known issue that breaks existing clients. Could you restore the custom object patch behavior following https://github.com/kubernetes-client/python/blob/master/devel/release.md#hot-issues?

Reply:
Every time I think I've finally learned how to read I prove myself wrong